meson.build

```diff
@@ -281,6 +281,7 @@ sources = [
   'nvme-wrap.c',
   'plugin.c',
   'libnvme-wrap.c',
+  'plugins/transcend/transcend-nvme.c'
```

Comment:
This is in the wrong `meson.build`. It should be added to `./plugins/meson.build.`